### ISSUE TITLE
targets command does not work

### ISSUE THREAD

**Issue by ysoldak** [tinygo-org/tinygo#3127]
On current "dev" branch head.

This also affects VSCode plugin, can not list and select target.

```
$ tinygo targets
...
esp32-mini32
esp32
panic: duplicate value 'esp32c3' in field : BuildTags

goroutine 1 [running]:
github.com/tinygo-org/tinygo/compileopts.(*TargetSpec).overrideProperties(0xc000374000, 0xc00036db80)
...
```

**Comment by sago35** [comment]
#3115

**Comment by deadprogram** [comment]
Released as part of `v0.26.0`
